src/app.js

```diff
@@ -1 +1,67 @@
+/* eslint-disable no-console */
 'use strict';
+
+const readline = require('readline');
+
+const input = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let bulls = 0;
+let cows = 0;
+
+const getRandomNumber = () => {
+  let result = '';
+
+  while (result.length < 4) {
+    const randomNumber = Math.floor(Math.random() * 10);
+
+    if (!result.includes(randomNumber)) {
+      result += randomNumber;
+    }
+  }
+
+  return result;
+};
+
+const randomNum = getRandomNumber();
+
+console.log(randomNum);
+
+const getBullsAndCows = (guess) => {
+  const guessArr = guess.split('');
+  const guessIsCorrect = guessArr.every((num, ind) => (
+    guessArr.lastIndexOf(num) === ind
+  ));
+
+  if (guessIsCorrect) {
+    guessArr.forEach((num, ind) => {
+      if (guessArr[ind] === randomNum[ind]) {
+        bulls += 1;
+      } else if (randomNum.includes(num)) {
+        cows += 1;
+      }
+    });
+    console.log(`bulls: ${bulls}, cows: ${cows}`);
+  } else {
+    console.log('enter 4 different numbers!');
+  }
```

Comment:
this way it seems clearer and cleaner to me, wdyt?
It is better to avoid nested `ifs` where possible
```suggestion
  if (!guessIsCorrect) {
    console.log('enter 4 different numbers!');
  }

   guessArr.forEach((num, ind) => {
     const isBull = guessArr[ind] === randomNum[ind];
     const isCow = randomNum.includes(num);
     
     if (isBull) {
       bulls += 1;
       return; // for breaking the current iteration of the `forEach`
      } 
      
     if (isCow) {
       cows += 1;
      }
    });
    
    console.log(`bulls: ${bulls}, cows: ${cows}`);
```